src/uu/factor/src/numeric.rs

```diff
@@ -8,8 +8,6 @@
 // * that was distributed with this source code.
 
 use std::mem::swap;
-use std::num::Wrapping;
-use std::u64::MAX as MAX_U64;
 
 pub fn gcd(mut a: u64, mut b: u64) -> u64 {
```

Comment:
could you please fix the clippy warning in this file?

Reply:
I had noticed the warning, but this is an unrelated linter bug: `gcd` is used in `milled_rabin` and `rho` (and I checked, removing the function cause a compilation error).

Should I add an `allow` attribute regardless?

Reply:
Oui, merci :)